wandb/sdk/launch/builder/build.py

```diff
@@ -334,20 +336,58 @@ def get_requirements_section(launch_project: LaunchProject, builder_type: str) -
         buildx_installed = False
     if launch_project.deps_type == "pip":
         requirements_files = []
-        if launch_project.project_dir is not None and os.path.exists(
-            os.path.join(launch_project.project_dir, "requirements.txt")
-        ):
+        pip_install_line = None
```

Comment:
nit: I wonder if there's some renaming that needs to be done here. This whole branch is in `pip` deps type, but its really `pip/poetry`. Then we have pip_install_line,.